frontend/src/components/LegalNotice/LegalNotice.jsx

```diff
@@ -4,106 +4,116 @@ import Mail from 'components/Footer/Mail';
-          </ul>
-        </div>
+    <Page
+      title="Mentions légales"
+      background="blue"
+    >
+      <div className={styles.container}>
+        <h1 className={styles.title} id="contactez-nous">
+          Contactez-nous
+        </h1>
+        <Mail
+          marge="null"
+        />
+        <h1 className={styles.title2} id="mentions-legales">
+          Mentions légales
+        </h1>
         <div>
-          <h3>Signaler un dysfonctionnement</h3>
-          <p className="fr-mb-2w">
-            Si vous rencontrez un défaut d’accessibilité vous empêchant d’accéder à un contenu
-            ou une fonctionnalité du site, merci de nous en faire part en
+          <div>
+            <h3 className={styles.sousTitre}>Éditeur de la Plateforme</h3>
+            <p className="fr-mb-2w">
+              La Plateforme
+              {config.appName}
+              , est éditée par l&lsquo;Incubateur de services numériques de la
+              Direction interministérielle du numérique (DINUM).
+            </p>
+            <p className="fr-mb-2w">
+              Coordonnées :
+              <br />
+              Adresse : DINUM, 20 avenue de Ségur, 75007 Paris
+              <br />
+              SIRET : 12000101100010 (secrétariat général du gouvernement)
+              <br />
+              SIREN : 120 001 011
+            </p>
+          </div>
+          <div>
+            <h3 className={styles.sousTitre}>Directeur de la publication</h3>
+            <p className="fr-mb-2w">
+              MINISTÈRE DE L&lsquo;ENSEIGNEMENT SUPÉRIEUR,
+              DE LA RECHERCHE ET DE L&lsquo;INNOVATION
+            </p>
+            <p className="fr-mb-2w">1 rue Descartes - 75231 Paris cedex 05</p>
+          </div>
+          <div>
+            <h3 className={styles.marginDiv}>Hébergement de la Plateforme</h3>
+            Ce site est hébergé en propre par Scalingo SAS, 15 avenue du Rhin, 67100 Strasbourg, France.
+          </div>
+          <div className={styles.marginDiv} id="accessibilite">
+            <h3 className={styles.sousTitre}>Accessibilité</h3>
+            <p className="fr-mb-2w">
+              La conformité aux normes d’accessibilité numérique est un objectif ultérieur.
+              En attendant, nous tâchons de rendre ce site
+              accessible à toutes et à tous :
+            </p>
+            <ul className={styles.liste}>
+              <li>Utilisation de composants accessibles (design system de l&lsquo;État)</li>
+              <li>Respect des bonnes pratiques (Pilida, Opquast...)</li>
+              <li>Tests manuels</li>
+            </ul>
+          </div>
+          <div>
+            <h3 className={styles.sousTitre}>Signaler un dysfonctionnement</h3>
+            <p className="fr-mb-2w">
+              Si vous rencontrez un défaut d’accessibilité vous empêchant d’accéder à un contenu
+              ou une fonctionnalité du site, merci de nous en faire part en
+              {' '}
+              <a
```

Comment:
should not be a mailto anymore but a redirection to our contact page

Reply:
I used a <Link to="/contact">
                nous contactons
              </Link>

Is that good for you ?

Reply:
<Link>

Reply:
I used a link instead to "/contact"

Reply:
i pushed the changes